### PR TITLE
gives the krebs a new sprite

### PR DESCRIPTION
**Pull request by ghost**
Why? New sprites are better, butter, in the pan two swipes code improved. That's how you do it.

Old crab sprites are still in the form of krebs.

Alt Text

### CODE REVIEW

**Comment by Armhulen** [comment]
new sprite = [kreb](https://tgstation13.org/phpBB/download/file.php?avatar=2738_1504978849.gif)

**Comment by ExcessiveUseOfCobblestone** [comment]
Did you replace the old crabs sprite or add this? If it's the latter, the OLD SPRITE should be "kreb"

**Comment by Armhulen** [comment]
both

**Comment by Armhulen** [comment]
wait, i made the crabs the new sprite but added a new mob called kreb which i'm doing right now

**Comment by ExcessiveUseOfCobblestone** [comment]
Lol

**Comment by PraiseRatvar** [comment]
the old sprite should b e called KREB

**Comment by Armhulen** [comment]
we don't call fregs FREG so we won't call krebs KREB

File: code/modules/mob/living/simple_animal/friendly/crab.dm [hunk 1]
```diff
@@ -43,4 +43,35 @@
 	response_help  = "pets"
 	response_disarm = "gently pushes aside"
 	response_harm   = "stomps"
-	gold_core_spawnable = 0
\ No newline at end of file
+	gold_core_spawnable = 0
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: FALSE

File: code/modules/mob/living/simple_animal/friendly/crab.dm [hunk 2]
```diff
@@ -43,4 +43,35 @@
...
+	name = "Evil Crab"
+	real_name = "Evil Crab"
+	desc = "Unnerving, isn't it? It has to be planning something nefarious..."
+	icon_state = "evilcrab"
+	icon_living = "evilcrab"
+	icon_dead = "evilcrab_dead"
+	response_help = "pokes"
+	response_disarm = "shoves"
+	response_harm = "stomps"
+	gold_core_spawnable = 1
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: TRUE

File: code/modules/mob/living/simple_animal/friendly/crab.dm [hunk 3]
```diff
@@ -43,4 +43,35 @@
...
+	name = "Kreb"
+	desc = "This is a real crab. The other crabs are simply gubbucks in disguise!
+	real_name = "Kreb"
+	icon_state = "kreb"
+	icon_living = "kreb"
+	icon_dead = "kreb_dead"
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm   = "stomps"
+	gold_core_spawnable = 0
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: FALSE

File: code/modules/mob/living/simple_animal/friendly/crab.dm [hunk 4]
```diff
@@ -43,4 +43,35 @@
...
+	icon_living = "evilcrab"
+	icon_dead = "evilcrab_dead"
+	response_help = "pokes"
+	response_disarm = "shoves"
+	response_harm = "stomps"
+	gold_core_spawnable = 1
+
+/mob/living/simple_animal/crab/kreb
+	name = "Kreb"
+	desc = "This is a real crab. The other crabs are simply gubbucks in disguise!
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: Forgot the quote

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Armhulen** [comment]
fixed!

**Comment by Armhulen** [comment]
fuck why didn't this work??

File: code/modules/mob/living/simple_animal/friendly/crab.dm [hunk 5]
```diff
@@ -43,4 +43,35 @@
...
+	icon_living = "evilcrab"
+	icon_dead = "evilcrab_dead"
+	response_help = "pokes"
+	response_disarm = "shoves"
+	response_harm = "stomps"
+	gold_core_spawnable = TRUE
+
+/mob/living/simple_animal/crab/kreb
+	name = "Kreb"
+	desc = "This is a real crab. The other crabs are simply gubbucks in disguise!
```
Comment (PraiseRatvar) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: ...disguise!" your forgot the BLIMMIN' QUOTATION MARK, CAN YOU NOT READ COMPILE LOGS ARM!!! ARM!!! AAAAAAAAAARM!!!!!!!!

**Comment by ExcessiveUseOfCobblestone** [comment]
if you click the x beside a commit it will tell you what goofed.

**Comment by Armhulen** [comment]
well DUH! i'll fix it later today

**Comment by Armhulen** [comment]
FIXED! DONE! GOOD!

**Comment by Anonmare** [comment]
snibbeti snab : DDD

**Comment by Bawhoppen** [comment]
this shouldn't have been as difficult as it was

**Review by Cyberboss — APPROVED** [review]

**Comment by Armhulen** [comment]
I'm the new guy on the block ok??

**Comment by Kerbin-Fiber** [comment]
TG now has krebs

**Comment by Qbopper** [comment]
it's kreb and freg not KREB or FREG

tangentially related: I did the freg thing like twice a few months ago how the fuck did it catch on at all

**Comment by Armhulen** [comment]
@qbopper it all started when I made a dragon and gave it the leader skin named freg

**Comment by Qbopper** [comment]
oh right I was there for that wasn't I

or maybe that was later

**Comment by optimumtact** [comment]
There is really no reason to keep the old sprite

**Comment by Armhulen** [comment]
it's not a terrible sprite and we have shameberos in game right now, which is literally just an admin meme

**Comment by optimumtact** [comment]
There is really no reason to keep the old sprite

**Comment by Armhulen** [comment]
fine, sorry qbop

**Comment by optimumtact** [comment]
There is really no reason to keep the old sprite

**Comment by Militaires** [comment]
the old one has legacy value keep it in

**Comment by optimumtact** [comment]
There is really no reason to keep the old sprite

**Comment by WJohn** [comment]
Very happy with these. 👍

**Pull request merged**